### DISCUSSION TITLE
Auto-refresh takes about 180 seconds.

### DISCUSSION THREAD

**Discussion by HitanshShah** [alshedivat/al-folio#2316] [Q&A] [answered]
Hi there! First of all, thanks a lot for this beautiful portfolio template. I have the website working locally, however, every time I save a new change, it builds the entire site and takes about 3 mins or more. I'm assuming it gets stuck on main.css since it is heavy, but is there a way this can be avoided? It's like even if I make 1 little change, it takes 3-4 mins just to see if the change is fine or if I need to redo it. Also, another issue I'm facing is that I changed the profile picture image in the folder, however it still shows the old one, which is weird because that one no longer exists in the folder (and this is happening with all images that I have replaced, keeping the name same).

**Comment by george-gca** [comment]
Please take a look at #2303

**Comment by george-gca** [comment, marked as answer]
Fixed in #2336.
